docs/src/installation/deployment.md

```diff
@@ -574,11 +574,68 @@ If you decided to set up a dex server, you can parametrize the [metal role](http
+- provide dhcp management addresses for management spine, management server and ipmi interface of the management server
+- Hairpin-NAT for the management server to access itself via its puplic IP, needed by the gitlab-runner to delegate CI-Jobs.
+- propagate a default gateway via BGP
 
-Documentation will follow soon. Please come to our Slack channel when you have come to this point and require any help.
+### Management Servers
+
+The second bastion hosts are the management servers. They are the main bootstrapping components of the management network. They also act as jump hosts for all components in a partition. Once they are installed and deployed, we are able to bootstrap all the other components. To bootstrap the management servers, we generate an ISO image which will automatically install an OS and an ansible user with ssh keys. It is preconfigured with a preseed file to allow an unattended OS installation for our needs. This is why we need remote access to the IPMI interface of the Management Servers: The generated ISO is attached via the virtual media function of the BMC. Ater that, all we have to do is boot from that virtual CD-ROM and wait for the installation to finish. Deployment jobs (Gitlab-CI) in a partition are delegated to the appropriate Management Servers, therefore we need a gitlabci-runner active on each management server.
+After the gitlab CI runner has been installed, you can trigger your Playbooks from the Gitlab-CI. The Ansible-Playbooks have to make sure that these functionalities are present on the Management Servers:
+
+- BMC proxy / BMC reverse proxy
+- Prometheus and exporters
+- gitlab-runner
+- bmc-catcher
+- image-cache
+- simple webserver to provide images
+- Onieboot and ZTP for cumulus switches
+- DHCP addresses for ipmi interfaces of the workers
+- DHCP addresses for cumulus switches
+
+### Management Spines
+
+The purpose of these switches is to connect the management interfaces of all switches to the management servers. The management spine's own management interface is connected to the management firewall for the bootstraping of the management spine itself. The management firewall will provide a DHCP address and DHCP options to start Cumulus' [Zero Touch Provisioning](https://docs.cumulusnetworks.com/cumulus-linux-42/Installation-Management/Zero-Touch-Provisioning-ZTP/); the images for all switches are downloaded from the management server (minio/webserver).
+Each management leaf is connected to both management spines to provide redundant connectivity to both management servers. BGP is used as a routing protocol so that, when a link goes down, an alternate path is used.
+In the picture above you can see that there are also switch management interfaces connected to the management spine. This has to be done so that we can bootstrap these switches; the management spine relays the DHCP requests from these switches to the management servers so that they are able to ONIE boot and get their ZTP scripts.
```

Comment:
```suggestion
In the picture above you can see that there are also switch management interfaces connected to the management spine. This is mandatory in order to bootstrap these switches; the management spine relays the DHCP requests from these switches to the management servers so that they are able to ONIE boot and get their ZTP scripts.
```

Reply:
In another paragraph you wrote onieboot, here it's ONIE boot